packages/auth/src/OAuth/oauthStorage.ts

```diff
@@ -22,16 +22,16 @@ export const getState = () => {
 };
 
 export const setPKCE = (private_key: string) => {
-	window.sessionStorage.setItem('ouath_pkce_key', private_key);
+	window.sessionStorage.setItem('oauth_pkce_key', private_key);
 };
 
 export const getPKCE = () => {
-	const ouath_pkce_key = window.sessionStorage.getItem('ouath_pkce_key');
-	window.sessionStorage.removeItem('ouath_pkce_key');
-	return ouath_pkce_key;
+	const oauth_pkce_key = window.sessionStorage.getItem('oauth_pkce_key');
```

Comment:
One approach is maintaining backwards-compatibility for reading from the incorrect key as a fall-back, while future reads/writes use the correct key:

```suggestion
	const oauth_pkce_key = window.sessionStorage.getItem('oauth_pkce_key') || window.sessionStorage.getItem('ouath_pkce_key');
	window.sessionStorage.removeItem('oauth_pkce_key');
	window.sessionStorage.removeItem('ouath_pkce_key');
```

Thoughts?

Reply:
@ericclemmons, that's awesome and clever 😉

Reply:
@Amplifiyer We may not need this based on @sprohaska's comment:
> https://github.com/aws-amplify/amplify-js/pull/4532#issuecomment-571058126